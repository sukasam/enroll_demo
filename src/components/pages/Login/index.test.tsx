import { render } from "@testing-library/react";
import Landing from "./index";

jest.mock("Services/mixpanel/initializeMixPanel", () => ({
    trackPageView: jest.fn()
}));

jest.mock("./components/LoginContent", () => {
    function LoginContent(): JSX.Element {
        return <div data-testid="login-content" />;
    }
    return LoginContent;
});

jest.mock("../Home/components/Aside", () => {
    function Aside(): JSX.Element {
        return <div data-testid="aside" />;
    }
    return Aside;
});

describe("Landing", () => {
    it("renders correctly", () => {
        const { container } = render(<Landing />);
        expect(container).toMatchSnapshot();
    });
});
