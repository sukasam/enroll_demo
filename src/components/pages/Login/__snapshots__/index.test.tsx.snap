// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders correctly 1`] = `
<div>
  <div
    class=MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 css-XXXXX-MuiGrid-root
  >
    <div
      class=MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-8 css-XXXXX-MuiGrid-root
    >
      <div
        data-testid="login-content"
      />
    </div>
    <div
      class=MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 aside-wrapper css-XXXXX-MuiGrid-root
      data-testid="starting_business_just_become_easier_section"
    >
      <div
        data-testid="aside"
      />
    </div>
  </div>
</div>
`;
